### PR TITLE
prefix_write not work with "/"

### PR DESCRIPTION
**Pull request by Bryce-huang**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Solutions

https://github.com/mosn/mosn/issues/1824

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

**Comment by Bryce-huang** [comment]
https://github.com/mosn/mosn/issues/1824

File: pkg/router/base_rule.go [hunk 1]
```diff
@@ -282,7 +282,7 @@ func (rri *RouteRuleImplBase) finalizePathHeader(ctx context.Context, headers ap
 
 		//If both prefix_rewrite and regex_rewrite are configured
 		//prefix rewrite by default
-		if len(rri.prefixRewrite) > 1 {
+		if len(rri.prefixRewrite) != 0 {
```
Comment (taoyuanyuan) [inline pkg/router/base_rule.go:285]: pls fix regex too.

**Review by taoyuanyuan — APPROVED** [review]
LGTM

**Review by nejisama — APPROVED** [review]
lgtm

**Pull request merged**